WriteVHDLSyntax.py

```diff
@@ -1233,6 +1329,46 @@ def writeProcCombination(module, str_ctrl_func, str_ports):
     # FIXME needs fixing to include template parameters for generic proc module writing
     """
     module_str = ""
+
+    if "PC_" in module.inst:
+        module_str += "  " + module.inst + "_mem_reader : entity work.mem_reader\n"
+        module_str += "    generic map (\n"
+        module_str += "      RAM_WIDTH    => 73,\n"
```

Comment:
@aryd Instead of hard-coding this number, can it be gotten from the `module` object? Maybe something like:
```suggestion
        module_str += "      RAM_WIDTH    => " + str(module.upstreams[0].bitwidth) + ",\n"
```

Reply:
This has been fixed.